src/wallet/rpcwallet.h

```diff
@@ -10,6 +10,7 @@
 class CRPCTable;
 class CWallet;
 class JSONRPCRequest;
+class UniValue;
```

Comment:
Why?

Reply:
`UniValue` is needed below for `UniValue getaddressinfo...`